--- Cura-3.2.1/CMakeLists.txt.orig	2018-02-14 01:50:14.000000000 -0800
+++ Cura-3.2.1/CMakeLists.txt	2018-04-05 18:05:17.549652826 -0700
@@ -36,10 +36,20 @@
 
 find_package(PythonInterp 3.5.0 REQUIRED)
 
+if(NOT PYTHON_SITE_PACKAGES_DIR)
+	if(APPLE OR WIN32)
+		set(PYTHON_SITE_PACKAGES_DIR lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages
+			CACHE STRING "Directory to install Python bindings to")
+	else()
+		set(PYTHON_SITE_PACKAGES_DIR lib/python${PYTHON_VERSION_MAJOR}/dist-packages
+			CACHE STRING "Directory to install Python bindings to")
+	endif()
+endif()
+
 install(DIRECTORY resources
         DESTINATION ${CMAKE_INSTALL_DATADIR}/cura)
 install(DIRECTORY plugins
-        DESTINATION lib${LIB_SUFFIX}/cura)
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cura)
 if(NOT APPLE AND NOT WIN32)
     install(FILES cura_app.py
             DESTINATION ${CMAKE_INSTALL_BINDIR}
@@ -47,16 +57,16 @@
             RENAME cura)
     if(EXISTS /etc/debian_version)
         install(DIRECTORY cura
-            DESTINATION lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}/dist-packages
+            DESTINATION ${PYTHON_SITE_PACKAGES_DIR}
             FILES_MATCHING PATTERN *.py)
         install(FILES ${CMAKE_BINARY_DIR}/CuraVersion.py
-            DESTINATION lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}/dist-packages/cura)
+            DESTINATION ${PYTHON_SITE_PACKAGES_DIR}/cura)
     else()
         install(DIRECTORY cura
-            DESTINATION lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages
+            DESTINATION ${PYTHON_SITE_PACKAGES_DIR}
             FILES_MATCHING PATTERN *.py)
         install(FILES ${CMAKE_BINARY_DIR}/CuraVersion.py
-            DESTINATION lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages/cura)
+            DESTINATION ${PYTHON_SITE_PACKAGES_DIR}/cura)
     endif()
     install(FILES ${CMAKE_BINARY_DIR}/cura.desktop
             DESTINATION ${CMAKE_INSTALL_DATADIR}/applications)
