--- x/src/software/convert/main_convertLDRToHDR.cpp	2020-02-19 23:31:44.000000000 -0800
+++ x/src/software/convert/main_convertLDRToHDR.cpp	2020-02-19 23:31:56.013995389 -0800
@@ -288,7 +288,7 @@
     float offset[3];
     for(int i=0; i<3; ++i)
         offset[i] = std::abs(meanRecovered[i] - meanVal[i]);
-    ALICEVISION_LOG_INFO("Offset between target source image and recovered from hdr = " << offset);
+    ALICEVISION_LOG_INFO("Offset between target source image and recovered from hdr = " << offset[0] << ", " << offset[1] << ", " << offset[2]);
 }
 
 
