--- a/MANIFEST.in	2018-12-05 10:17:17.000000000 -0800
+++ a/MANIFEST.in	2019-03-14 10:38:05.846707166 -0700
@@ -5,7 +5,6 @@
 include CHANGES.md
 include Makefile
 include setup.py
-include setup.cfg
 
 include bin/markdown2
 include test/api.doctests
--- a/setup.py	2018-12-05 10:17:17.000000000 -0800
+++ a/setup.py	2019-03-14 10:38:05.229723280 -0700
@@ -3,7 +3,8 @@
 import os
 import sys
 
-from setuptools import setup
+import distutils
+from distutils.core import setup
 
 _top_dir = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, os.path.join(_top_dir, "lib"))
--- a/setup.cfg	2019-05-29 11:55:44.000000000 -0700
+++ a/setup.cfg	2019-09-09 21:06:46.820028174 -0700
@@ -1,2 +0,0 @@
-[wheel]
-universal = 1
