--- sabnzbd/growler.py.orig	2016-03-15 19:25:51.000000000 +0000
+++ sabnzbd/growler.py	2016-04-08 12:21:10.115977429 +0100
@@ -37,7 +37,13 @@
 from sabnzbd.encoding import unicoder
 from sabnzbd.constants import NOTIFY_KEYS
 
-from gntp import GNTPRegister
+try:
+    # <= version 0.6
+    from gntp import GNTPRegister
+except ImportError:
+    # >= version 1.0
+    from gntp.core import GNTPRegister
+
 from gntp.notifier import GrowlNotifier
 try:
     import Growl
