--- SABnzbd-3.0.0/sabnzbd/rss.py	2020-08-18 19:41:02.347638475 -0700
+++ SABnzbd-3.0.0/sabnzbd/rss.py	2020-08-18 19:51:24.017651913 -0700
@@ -274,7 +274,7 @@
         first = (feed not in self.jobs) and ignoreFirst
 
         # Add SABnzbd's custom User Agent
-        feedparser.USER_AGENT = "SABnzbd/%s" % sabnzbd.__version__
+        feedparser.api.USER_AGENT = "SABnzbd/%s" % sabnzbd.__version__
 
         # Read the RSS feed
         msg = None
@@ -283,8 +283,8 @@
             all_entries = []
             for uri in uris:
                 uri = uri.replace(" ", "%20")
-                logging.debug("Running feedparser on %s", uri)
-                feed_parsed = feedparser.parse(uri.replace("feed://", "http://"))
+                logging.debug("Running feedparser.api.on %s", uri)
+                feed_parsed = feedparser.api.parse(uri.replace("feed://", "http://"))
                 logging.debug("Done parsing %s", uri)
 
                 if not feed_parsed:
@@ -641,11 +641,11 @@
     """ Apply options that work for SABnzbd
         Add additional parsing of attributes
     """
-    feedparser.SANITIZE_HTML = 0
-    feedparser.PARSE_MICROFORMATS = 0
+    feedparser.api.SANITIZE_HTML = 0
+    feedparser.api.PARSE_MICROFORMATS = 0
 
     # Add our own namespace
-    feedparser._FeedParserMixin.namespaces["http://www.newznab.com/DTD/2010/feeds/attributes/"] = "newznab"
+    feedparser.api._FeedParserMixin.namespaces["http://www.newznab.com/DTD/2010/feeds/attributes/"] = "newznab"
 
     # Add parsers for the namespace
     def _start_newznab_attr(self, attrsD):
@@ -659,14 +659,14 @@
             context["newznab"][attrsD["name"]] = attrsD["value"]
             # Try to get date-object
             if attrsD["name"] == "usenetdate":
-                context["newznab"][attrsD["name"] + "_parsed"] = feedparser._parse_date(attrsD["value"])
+                context["newznab"][attrsD["name"] + "_parsed"] = feedparser.api._parse_date(attrsD["value"])
         except KeyError:
             pass
 
-    feedparser._FeedParserMixin._start_newznab_attr = _start_newznab_attr
-    feedparser._FeedParserMixin._start_nZEDb_attr = _start_newznab_attr
-    feedparser._FeedParserMixin._start_nzedb_attr = _start_newznab_attr
-    feedparser._FeedParserMixin._start_nntmux_attr = _start_newznab_attr
+    feedparser.api._FeedParserMixin._start_newznab_attr = _start_newznab_attr
+    feedparser.api._FeedParserMixin._start_nZEDb_attr = _start_newznab_attr
+    feedparser.api._FeedParserMixin._start_nzedb_attr = _start_newznab_attr
+    feedparser.api._FeedParserMixin._start_nntmux_attr = _start_newznab_attr
 
 
 def _HandleLink(
@@ -772,7 +772,7 @@
         except:
             pass
     finally:
-        # We need to convert it to local timezone, feedparser always returns UTC
+        # We need to convert it to local timezone, feedparser.api.always returns UTC
         age = age - datetime.timedelta(seconds=time.timezone)
 
     # Maybe the newznab also provided SxxExx info
