--- src/vars	2004-12-07 03:42:48.000000000 -0800
+++ src/vars	2004-12-07 03:42:48.000000000 -0800
@@ -1,10 +1,19 @@
 
-TARGET = kegsmac
-OBJECTS = $(OBJECTS1) macsnd_driver.o macdriver.o
-CCOPTS = -O2 -DMAC
+TARGET = xkegs
+OBJECTS = $(OBJECTS1) xdriver.o
+CC = gcc
+CCOPTS = -O2 -Wall -fomit-frame-pointer -march=native
+OPTS = -DKEGS_LITTLE_ENDIAN
 SUFFIX =
-NAME = kegsmac
+NAME = xkegs
+LDFLAGS =
+LDOPTS =
+LD = $(CC)
+EXTRA_LIBS = -lXext
+EXTRA_SPECIALS =
 
-XOPTS = -Wall -fpascal-strings -mdynamic-no-pic -arch ppc
-XLIBS =
+AS = cc
+PERL = perl
+
+XOPTS = -I/usr/X11R6/include
 
