--- KiKit-1.0.5/setup.py	2022-04-07 23:37:01.000000000 -0700
+++ KiKit-1.0.5/setup.py	2022-05-05 20:47:25.884324440 -0700
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 
 import setuptools
-import versioneer
 import os
 
 try:
@@ -25,8 +24,8 @@
 setuptools.setup(
     name="KiKit",
     python_requires='>=3.7',
-    version=versioneer.get_version(),
-    cmdclass=versioneer.get_cmdclass(),
+    version='1.05',
+    cmdclass={},
     author="Jan Mrázek",
     author_email="email@honzamrazek.cz",
     description="Automation for KiCAD boards",
@@ -50,7 +49,6 @@
         "commentjson>=0.9"
     ],
     setup_requires=[
-        "versioneer"
     ],
     extras_require={
         "dev": ["pytest"],
