--- lib/gitmon/gitmon.conf.example~	2011-04-21 00:46:08.000000000 -0700
+++ lib/gitmon/gitmon.conf.example	2018-02-08 08:44:27.228369569 -0800
@@ -4,17 +4,8 @@
 # Notifier type #
 #################
 
-#Notifier implementation. Possible choices: 'command.line', 'growl'
-notifier.type = growl
-
-##########################
-# Growl Notifier options #
-##########################
-
-#These options are relevant to notifier.type = growl
-
-#Keep notifications on screen untill you close them 
-growl.sticky.notifications = 0
+#Notifier implementation. Possible choices: 'command.line'
+notifier.type = command.line
 
 #################################
 # Command Line Notifier options #
--- lib/gitmon/notifiers.py~	2011-04-21 00:46:08.000000000 -0700
+++ lib/gitmon/notifiers.py	2018-02-08 08:46:10.769363995 -0800
@@ -18,7 +18,6 @@
 """
 
 import subprocess
-import Growl
 
 class Notifier(object):
 
@@ -33,8 +32,6 @@
         """Returns singleton instance of given notifier type"""
         if type == 'command.line':
             return CommandLineNotifier.instance(config)
-        if type == 'growl':
-            return GrowlNotifier.instance(config)
 
 class CommandLineNotifier(Notifier):
 
@@ -64,26 +61,3 @@
         if retcode != 0:
             print 'Error while notifying: %s, %s' % (retcode, args)
 
-class GrowlNotifier(Notifier):
-
-    inst = None
-   
-    @classmethod
-    def instance(cls, config):
-        if not GrowlNotifier.inst:
-            GrowlNotifier.inst = GrowlNotifier(config)
-        return GrowlNotifier.inst
-
-    def notify(self, title, message, image, cwd):
-        if image:
-            image = Growl.Image.imageFromPath(image)
-        sticky = bool(int(self.config['growl.sticky.notifications']))
-        growl = Growl.GrowlNotifier(applicationName='GitMon', \
-                applicationIcon=image, \
-                notifications=['update'], \
-                defaultNotifications=['update'])
-        if not hasattr(self, 'registered'):
-            growl.register()
-            self.registered = True
-        growl.notify('update', title, message, icon=image, sticky=sticky)
-        
--- lib/gitmon/gitmon.py~	2011-04-21 00:46:08.000000000 -0700
+++ lib/gitmon/gitmon.py	2018-02-08 08:45:05.406367514 -0800
@@ -446,7 +446,7 @@
         notifier.notify(title, message, image, repo.path_full)
 
     def selftest(self):
-        print "GitMon Self Test. You should see a notification right now. Install Growl from http://growl.info if you don't."
+        print "GitMon Self Test. You should see a notification right now."
         notifier = Notifier.create(notifier_type, self.config)
         notifier.notify('GitMon Test', 'It Works!', gitmon_dir + '/git.png', gitmon_dir)
 
