--- a/filetype.egg-info/SOURCES.txt	2019-10-23 12:43:18.000000000 -0700
+++ a/filetype.egg-info/SOURCES.txt	2019-10-23 12:45:35.000000000 -0700
@@ -4,10 +4,6 @@
 README.rst
 setup.cfg
 setup.py
-examples/__init__.py
-examples/buffer.py
-examples/bytes.py
-examples/file.py
 filetype/__init__.py
 filetype/filetype.py
 filetype/helpers.py
--- a/filetype.egg-info/top_level.txt	2019-10-23 12:49:51.000000000 -0700
+++ a/filetype.egg-info/top_level.txt	2019-10-23 12:51:15.000000000 -0700
@@ -1,2 +1 @@
-examples
 filetype
--- a/examples/buffer.py	2019-10-23 12:56:37.788675212 -0700
+++ a/examples/buffer.py	1969-12-31 16:00:00.000000000 -0800
@@ -1,23 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-from __future__ import absolute_import
-
-import filetype
-
-
-def main():
-    f = open('tests/fixtures/sample.jpg', 'rb')
-    data = f.read()
-
-    kind = filetype.guess(data)
-    if kind is None:
-        print('Cannot guess file type!')
-        return
-
-    print('File extension: %s' % kind.extension)
-    print('File MIME type: %s' % kind.mime)
-
-
-if __name__ == '__main__':
-    main()
--- a/examples/bytes.py	2019-10-23 12:56:37.788675212 -0700
+++ a/examples/bytes.py	1969-12-31 16:00:00.000000000 -0800
@@ -1,22 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-from __future__ import absolute_import
-
-import filetype
-
-
-def main():
-    buf = bytearray([0xFF, 0xD8, 0xFF, 0x00, 0x08])
-    kind = filetype.guess(buf)
-
-    if kind is None:
-        print('Cannot guess file type!')
-        return
-
-    print('File extension: %s' % kind.extension)
-    print('File MIME type: %s' % kind.mime)
-
-
-if __name__ == '__main__':
-    main()
--- a/examples/file.py	2019-10-23 12:56:37.788675212 -0700
+++ a/examples/file.py	1969-12-31 16:00:00.000000000 -0800
@@ -1,20 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-from __future__ import absolute_import
-
-import filetype
-
-
-def main():
-    kind = filetype.guess('tests/fixtures/sample.jpg')
-    if kind is None:
-        print('Cannot guess file type!')
-        return
-
-    print('File extension: %s' % kind.extension)
-    print('File MIME type: %s' % kind.mime)
-
-
-if __name__ == '__main__':
-    main()
--- a/examples/__init__.py	2019-10-23 12:56:37.788675212 -0700
+++ a/examples/__init__.py	1969-12-31 16:00:00.000000000 -0800
@@ -1 +0,0 @@
-# -*- coding: utf-8 -*-
