--- ocl/build_kernel.c~	2017-01-17 10:10:13.000000000 -0800
+++ ocl/build_kernel.c	2018-01-23 22:23:46.778629296 -0800
@@ -15,6 +15,12 @@
     f = fopen(fullpath, "rb");
   }
   if (!f) {
+    /* Then try from /usr/lib/sgminer-nh */
+    snprintf(fullpath, PATH_MAX, "/usr/lib/sgminer-nh/%s", sgminer_path, filename);
+    applog(LOG_DEBUG, "Trying to open %s...", fullpath);
+    f = fopen(fullpath, "rb");
+  }
+  if (!f) {
     /* Then try from the path sgminer was called */
     snprintf(fullpath, PATH_MAX, "%s/%s", sgminer_path, filename);
     applog(LOG_DEBUG, "Trying to open %s...", fullpath);
