--- a/CMakeLists.txt	2019-03-20 21:57:00.000000000 -0700
+++ a/CMakeLists.txt	2019-03-20 21:59:18.010677514 -0700
@@ -15,10 +15,6 @@
     add_definitions(-DARCUS)
 endif ()
 
-#For reading image files.
-find_package(Stb REQUIRED)
-include_directories(${Stb_INCLUDE_DIRS})
-
 option(USE_SYSTEM_LIBS "Use the system libraries if available" OFF)
 if(USE_SYSTEM_LIBS)
     find_package(RapidJSON CONFIG REQUIRED)
