--- configure.ac~	2017-01-17 10:10:13.000000000 -0800
+++ configure.ac	2018-01-22 23:08:51.896184419 -0800
@@ -255,7 +255,7 @@
 AM_CONDITIONAL([HAVE_x86_64], [test x$have_x86_64 = xtrue])
 
 AC_CONFIG_SUBDIRS([submodules/jansson])
-JANSSON_LIBS="submodules/jansson/src/.libs/libjansson.a"
+JANSSON_LIBS="-ljansson"
 
 PKG_PROG_PKG_CONFIG()
 
--- Makefile.am~	2017-01-17 10:10:13.000000000 -0800
+++ Makefile.am	2018-01-22 23:08:51.176191183 -0800
@@ -8,11 +8,11 @@
 		  API.class API.java api-example.c hexdump.c \
 		  doc/API doc/FAQ doc/GPU doc/SCRYPT doc/windows-build.txt
 
-SUBDIRS		= lib submodules ccan sph
+SUBDIRS		= lib ccan sph
 
 bin_PROGRAMS     = sgminer
 
-sgminer_CPPFLAGS = $(PTHREAD_FLAGS) -std=gnu99 $(JANSSON_CPPFLAGS)
+sgminer_CPPFLAGS = $(PTHREAD_FLAGS) -std=gnu99
 sgminer_LDFLAGS  = $(PTHREAD_FLAGS)
 sgminer_LDADD    = $(DLOPEN_FLAGS) @LIBCURL_LIBS@ @JANSSON_LIBS@ @PTHREAD_LIBS@ \
 		  @OPENCL_LIBS@ @NCURSES_LIBS@ @PDCURSES_LIBS@ @WS2_LIBS@ \
