--- a/setup.py	2021-05-28 02:27:10.000000000 -0700
+++ a/setup.py	2022-12-29 21:03:12.315645674 -0800
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 
 import setuptools
-import versioneer
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
@@ -9,8 +8,8 @@
 setuptools.setup(
     name="pcbnewTransition",
     python_requires='>3.7',
-    version=versioneer.get_version(),
-    cmdclass=versioneer.get_cmdclass(),
+    version='0.3.0',
+    cmdclass={},
     author="Jan Mrázek",
     author_email="email@honzamrazek.cz",
     description="Library that allows you to support both, KiCAD 5 and KiCAD 6 in your plugins",
@@ -24,7 +23,6 @@
         "Operating System :: OS Independent",
     ],
     setup_requires=[
-        "versioneer"
     ],
     zip_safe=False,
     include_package_data=True
