--- desktop/MediaElch.desktop~	2015-02-02 11:55:13.000000000 -0800
+++ desktop/MediaElch.desktop	2015-04-03 09:11:30.340835407 -0700
@@ -1,9 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
-Version=2.2.2
 Type=Application
-Terminal=false
+Comment=Metadata downloader for video files
 Exec=/usr/bin/MediaElch
 Name=MediaElch
 Icon=/usr/share/pixmaps/MediaElch.png
-Categories=Video;
+Categories=Qt;AudioVideo;Video;
