--- a/configure
+++ b/configure
@@ -5907,7 +5907,7 @@
 enabled pic && enable_weak_pic
 
 is_qmake5(){
-    $1 --version 2>&1 | egrep -q -e "Qt version 5\.[2-9]\.[0-9]"
+    $1 --version 2>&1 | egrep -q -e 'Qt version 5\.(([1-9][0-9]+)|[2-9])\.[0-9]'
 }
 
 # qmake-qt5 /usr/lib64/qt5/bin/qmake /usr/lib/x86_64-linux-gnu/qt5/bin/qmake
--- a/libs/libmythbase/libmythbase.pro
+++ b/libs/libmythbase/libmythbase.pro
@@ -154,7 +154,7 @@
 
 QT += xml sql network
 
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/libs/libmyth/libmyth.pro
+++ b/libs/libmyth/libmyth.pro
@@ -1,7 +1,7 @@
 include ( ../../settings.pro )
 
 QT += network xml sql script
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 android: QT += androidextras
 }
@@ -169,7 +169,7 @@
     SOURCES += mediamonitor-unix.cpp
     HEADERS += mediamonitor-unix.h
     !android {
-    contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+    contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
         using_qtdbus: QT += dbus
     } else {
         using_qtdbus: CONFIG += qdbus
--- a/libs/libmyth/test/test_audioconvert/test_audioconvert.pro
+++ b/libs/libmyth/test/test_audioconvert/test_audioconvert.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmyth/test/test_audioutils/test_audioutils.pro
+++ b/libs/libmyth/test/test_audioutils/test_audioutils.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmyth/test/test_programinfo/test_programinfo.pro
+++ b/libs/libmyth/test/test_programinfo/test_programinfo.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythbase/test/test_mythsystem/test_mythsystem.pro
+++ b/libs/libmythbase/test/test_mythsystem/test_mythsystem.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythbase/test/test_mythsystemlegacy/test_mythsystemlegacy.pro
+++ b/libs/libmythbase/test/test_mythsystemlegacy/test_mythsystemlegacy.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythbase/test/test_mythtimer/test_mythtimer.pro
+++ b/libs/libmythbase/test/test_mythtimer/test_mythtimer.pro
@@ -3,7 +3,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythbase/test/test_template/test_template.pro
+++ b/libs/libmythbase/test/test_template/test_template.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythmetadata/libmythmetadata.pro
+++ b/libs/libmythmetadata/libmythmetadata.pro
@@ -1,7 +1,7 @@
 include ( ../../settings.pro )
 
 QT += network xml sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/libs/libmythmetadata/test/test_videometadata/test_videometadata.pro
+++ b/libs/libmythmetadata/test/test_videometadata/test_videometadata.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythprotoserver/libmythprotoserver.pro
+++ b/libs/libmythprotoserver/libmythprotoserver.pro
@@ -1,7 +1,7 @@
 include ( ../../settings.pro )
 
 QT += network sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/libs/libmythtv/libmythtv.pro
+++ b/libs/libmythtv/libmythtv.pro
@@ -1,7 +1,7 @@
 include ( ../../settings.pro )
 
 QT += network xml sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/libs/libmythtv/test/test_copyframes/test_copyframes.pro
+++ b/libs/libmythtv/test/test_copyframes/test_copyframes.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythtv/test/test_eitfixups/test_eitfixups.pro
+++ b/libs/libmythtv/test/test_eitfixups/test_eitfixups.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythtv/test/test_iptvrecorder/test_iptvrecorder.pro
+++ b/libs/libmythtv/test/test_iptvrecorder/test_iptvrecorder.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythtv/test/test_mpegtables/test_mpegtables.pro
+++ b/libs/libmythtv/test/test_mpegtables/test_mpegtables.pro
@@ -5,7 +5,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 CONFIG += qtestlib
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += testlib
 }
 
--- a/libs/libmythui/libmythui.pro
+++ b/libs/libmythui/libmythui.pro
@@ -4,7 +4,7 @@
 contains(QT_VERSION, ^4\\.[0-9]\\..*) {
 QT += webkit
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 QT += webkitwidgets
 android: QT += androidextras
--- a/programs/mythavtest/mythavtest.pro
+++ b/programs/mythavtest/mythavtest.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro )
 
 QT += network xml sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythbackend/mythbackend.pro
+++ b/programs/mythbackend/mythbackend.pro
@@ -10,7 +10,7 @@
    # need the following for QUrl::addQueryItem
    DEFINES += QT_DISABLE_DEPRECATED_BEFORE
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythccextractor/mythccextractor.pro
+++ b/programs/mythccextractor/mythccextractor.pro
@@ -2,7 +2,7 @@
 include ( ../programs-libs.pro )
 
 QT += sql network
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythcommflag/mythcommflag.pro
+++ b/programs/mythcommflag/mythcommflag.pro
@@ -2,7 +2,7 @@
 include (../../version.pro)
 include ( ../programs-libs.pro )
 
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythfilerecorder/mythfilerecorder.pro
+++ b/programs/mythfilerecorder/mythfilerecorder.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro )
 
 QT += network xml sql script
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythfilldatabase/mythfilldatabase.pro
+++ b/programs/mythfilldatabase/mythfilldatabase.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro )
 
 QT += network xml sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythfrontend/mythfrontend.pro
+++ b/programs/mythfrontend/mythfrontend.pro
@@ -11,7 +11,7 @@
 QT += webkit
 using_qtdbus: CONFIG += qdbus
 }
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 QT += webkitwidgets
 using_qtdbus: QT += dbus
--- a/programs/mythjobqueue/mythjobqueue.pro
+++ b/programs/mythjobqueue/mythjobqueue.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro)
 
 QT += sql network
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythlcdserver/mythlcdserver.pro
+++ b/programs/mythlcdserver/mythlcdserver.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro )
 
 QT += network xml sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 using_opengl:QT += opengl
--- a/programs/mythlogserver/mythlogserver.pro
+++ b/programs/mythlogserver/mythlogserver.pro
@@ -2,7 +2,7 @@
 include ( ../programs-libs.pro )
 
 QT += sql network
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythmediaserver/mythmediaserver.pro
+++ b/programs/mythmediaserver/mythmediaserver.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro )
 
 QT += network sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythpreviewgen/mythpreviewgen.pro
+++ b/programs/mythpreviewgen/mythpreviewgen.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro )
 
 QT += network sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythscreenwizard/mythscreenwizard.pro
+++ b/programs/mythscreenwizard/mythscreenwizard.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro)
 
 QT += sql network
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythtranscode/mythtranscode.pro
+++ b/programs/mythtranscode/mythtranscode.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro)
 
 QT += network xml sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythtv-setup/mythtv-setup.pro
+++ b/programs/mythtv-setup/mythtv-setup.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro )
 
 QT += network xml sql
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythutil/mythutil.pro
+++ b/programs/mythutil/mythutil.pro
@@ -3,7 +3,7 @@
 include ( ../programs-libs.pro )
 
 QT += network sql xml
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
--- a/programs/mythwelcome/mythwelcome.pro
+++ b/programs/mythwelcome/mythwelcome.pro
@@ -3,7 +3,7 @@
 include (../programs-libs.pro)
 
 QT += xml sql network
-contains(QT_VERSION, ^5\\.[0-9]\\..*) {
+contains(QT_VERSION, ^5\\.(([1-9][0-9]+)|[0-9])\\..*) {
 QT += widgets
 }
 
