CUDA 9 SDK doesn't support Compute Model 2.0
--- Makefile.am~	2017-01-01 12:38:31.000000000 -0800
+++ Makefile.am	2018-01-20 21:33:31.380923498 -0800
@@ -137,8 +137,9 @@
 	$(NVCC) $(nvcc_FLAGS) -I cudpp-2.1/include --maxrregcount=80 -o $@ -c $<
 
 # This kernel need also an older SM to be able to autotune kernels
+# (but compute_20 is no longer available)
 scrypt/salsa_kernel.o: scrypt/salsa_kernel.cu
-	$(NVCC) $(nvcc_FLAGS) -gencode=arch=compute_20,code=\"sm_21,compute_20\" --maxrregcount=80 -o $@ -c $<
+	$(NVCC) $(nvcc_FLAGS) -gencode=arch=compute_30,code=\"sm_30,compute_30\" --maxrregcount=80 -o $@ -c $<
 	
 skein.o: skein.cu
 	$(NVCC) $(nvcc_FLAGS) --maxrregcount=64 -o $@ -c $<
