diff -urN gsport_0.31.orig/src/vars gsport_0.31/src/vars
--- src/vars	1969-12-31 16:00:00.000000000 -0800
+++ src/vars	2015-06-10 17:38:21.729464444 -0700
@@ -0,0 +1,17 @@
+TARGET = gsportx
+OBJECTS = $(OBJECTS1) $(TFEOBJ) $(ATOBJ) $(PCAPOBJ) xdriver.o
+CC = gcc
+CCOPTS = -O2 -Wall -fomit-frame-pointer -std=gnu99 -march=native -DHAVE_TFE -DHAVE_ATBRIDGE -DTOGGLE_STATUS
+OPTS = -DGSPORT_LITTLE_ENDIAN 
+SUFFIX =
+NAME = gsportx
+LDFLAGS =
+LDOPTS =
+LD = g++
+EXTRA_LIBS = -lXext -ldl
+EXTRA_SPECIALS =
+
+AS = cc
+PERL = perl
+
+XOPTS = -I/usr/X11R6/include
