--- Uranium-3.2.1/CMakeLists.txt.orig	2018-04-05 18:11:04.858594720 -0700
+++ Uranium-3.2.1/CMakeLists.txt	2018-04-05 18:14:24.055603092 -0700
@@ -45,14 +45,24 @@
 # Build Translations
 CREATE_TRANSLATION_TARGETS()
 
+if(NOT PYTHON_SITE_PACKAGES_DIR)
+	if(APPLE OR WIN32)
+		set(PYTHON_SITE_PACKAGES_DIR lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages
+			CACHE STRING "Directory to install Python bindings to")
+	else()
+		set(PYTHON_SITE_PACKAGES_DIR lib/python${PYTHON_VERSION_MAJOR}/dist-packages CACHE
+			STRING "Directory to install Python bindings to")
+	endif()
+endif()
+
 if(EXISTS /etc/debian_version)
-    install(DIRECTORY UM DESTINATION lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}/dist-packages)
+    install(DIRECTORY UM DESTINATION ${PYTHON_SITE_PACKAGES_DIR})
 else()
-    install(DIRECTORY UM DESTINATION lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages)
+    install(DIRECTORY UM DESTINATION ${PYTHON_SITE_PACKAGES_DIR})
 endif()
 install(FILES ${CMAKE_SOURCE_DIR}/cmake/UraniumTranslationTools.cmake
-        DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/Modules/ )
+        DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake/Modules/ )
 install(DIRECTORY resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
-install(DIRECTORY plugins DESTINATION lib${LIB_SUFFIX}/uranium)
+install(DIRECTORY plugins DESTINATION ${CMAKE_INSTALL_LIBDIR}/uranium)
 
 include(CPackConfig.cmake)
