--- x/cmake/FindOpenEXR.cmake	2019-07-12 01:05:21.000000000 -0700
+++ x/cmake/FindOpenEXR.cmake	2020-02-23 20:48:23.642467143 -0800
@@ -61,11 +61,11 @@
     # message(WARNING "Yes, file exist")
 
     # Must be at least 2.0
-    file(STRINGS "${OPENEXR_INCLUDE_PATH}/OpenEXR/OpenEXRConfig.h" TMP REGEX "^#define OPENEXR_VERSION_STRING .*$")
+    file(STRINGS "${OPENEXR_INCLUDE_PATH}/x86_64-pc-linux-gnu/OpenEXR/OpenEXRConfig.h" TMP REGEX "^#define OPENEXR_VERSION_STRING .*$")
     string (REGEX MATCHALL "[0-9]+[.0-9]+" OPENEXR_VERSION ${TMP})
-    file(STRINGS "${OPENEXR_INCLUDE_PATH}/OpenEXR/OpenEXRConfig.h" TMP REGEX "^#define OPENEXR_VERSION_MAJOR .*$")
+    file(STRINGS "${OPENEXR_INCLUDE_PATH}/x86_64-pc-linux-gnu/OpenEXR/OpenEXRConfig.h" TMP REGEX "^#define OPENEXR_VERSION_MAJOR .*$")
     string (REGEX MATCHALL "[0-9]+" OPENEXR_VERSION_MAJOR ${TMP})
-    file(STRINGS "${OPENEXR_INCLUDE_PATH}/OpenEXR/OpenEXRConfig.h" TMP REGEX "^#define OPENEXR_VERSION_MINOR .*$")
+    file(STRINGS "${OPENEXR_INCLUDE_PATH}/x86_64-pc-linux-gnu/OpenEXR/OpenEXRConfig.h" TMP REGEX "^#define OPENEXR_VERSION_MINOR .*$")
     string (REGEX MATCHALL "[0-9]+" OPENEXR_VERSION_MINOR ${TMP})
 else ()
     message(WARNING "No, default to 1.6.1")
