diff -ur concerto-1.9.3-NoCAS/admin/app/frontpage/admin.php concerto-1.9.3-NoCAS-patched/admin/app/frontpage/admin.php
--- concerto-1.9.3-NoCAS/admin/app/frontpage/admin.php	2010-03-14 17:10:28.000000000 -0700
+++ concerto-1.9.3-NoCAS-patched/admin/app/frontpage/admin.php	2011-10-14 13:42:12.020471971 -0700
@@ -62,7 +62,6 @@
 
 <h3>Admin Privs: <span class="emph"><?= isAdmin() ?></span></h3>
 <h3>Reset Session: <span class="emph"><a href="<?=ADMIN_URL?>/frontpage/su?r=1">reset</a></span></h3>
-<h3>Admin Revision: <span class="emph"><?system('svnversion')?></span></h3>
 <a href="<?= ADMIN_URL ?>/frontpage/phpinfo">PHP Info</a><br />
 <a href="<?= ADMIN_URL ?>/frontpage/mailer">Send Mail</a><br />
 <a href="<?= ADMIN_URL ?>/frontpage/addtemplate">Template Importer</a></br>
