--- a/setup.py	2020-03-03 07:28:51.000000000 -0800
+++ a/setup.py	2020-07-18 16:12:17.978069013 -0700
@@ -19,7 +19,7 @@
 
 # Requirements for setup.py
 SETUP_REQUIRES = [
-	"markdown>=3.1,<3.2",
+	"markdown>=3.1",
 ]
 
 # Requirements for our application
@@ -27,44 +27,44 @@
 	# the following dependencies are non trivial to update since later versions introduce backwards incompatible
 	# changes that might affect plugins, or due to other observed problems
 
-	"flask>=0.12,<0.13",         # newer versions require newer Jinja versions
-	"Jinja2>=2.8.1,<2.9",        # Jinja 2.9 has breaking changes WRT template scope - we can't
+	"flask>=0.12",         # newer versions require newer Jinja versions
+	"Jinja2>=2.8.1",        # Jinja 2.9 has breaking changes WRT template scope - we can't
 	                             # guarantee backwards compatibility for plugins and such with that
 	                             # version, hence we need to pin to a lower version for now. See #1697
-	"tornado==4.5.3",            # a memory leak was observed in tornado >= 5, see #2585
+	"tornado",            # a memory leak was observed in tornado >= 5, see #2585
 	"regex!=2018.11.6",          # avoid broken 2018.11.6. See #2874
 
 	# anything below this should be checked on releases for new versions
 
-	"Flask-Login>=0.4.1,<0.5",
-	"Flask-Babel>=0.12,<0.13",
-	"Flask-Assets>=0.12,<0.13",
-	"blinker>=1.4,<1.5",         # dependency of the now vendor bundled flask_principal
-	"werkzeug>=0.16,<0.17",
-	"cachelib>=0.1,<0.2",
-	"PyYAML>=5.1,<6",
-	"markdown>=3.1,<3.2",
-	"pyserial>=3.4,<3.5",
-	"netaddr>=0.7.19,<0.8",
-	"watchdog>=0.9.0,<0.10",
+	"Flask-Login>=0.4.1",
+	"Flask-Babel>=0.12",
+	"Flask-Assets>=0.12",
+	"blinker>=1.4",         # dependency of the now vendor bundled flask_principal
+	"werkzeug>=0.16",
+	"cachelib>=0.1",
+	"PyYAML>=5.1",
+	"markdown>=3.1",
+	"pyserial>=3.4",
+	"netaddr>=0.7.19",
+	"watchdog>=0.9.0",
 	"sarge==0.1.5post0",
-	"netifaces>=0.10.9,<0.11",
-	"pylru>=1.2,<1.3",
-	"rsa>=4.0,<5",
-	"pkginfo>=1.5.0.1,<1.6",
-	"requests>=2.22.0,<3",
-	"semantic_version>=2.8,<2.9",
-	"psutil>=5.6.5,<5.7",
-	"Click>=7,<8",
-	"awesome-slugify>=1.6.5,<1.7",
-	"feedparser>=5.2.1,<5.3",
-	"future>=0.18.2,<0.19",
-	"websocket-client>=0.56,<0.57",
-	"wrapt>=1.11.2,<1.12",
-	"emoji>=0.5.4,<0.6",
-	"frozendict>=1.2,<1.3",
-	"sentry-sdk==0.13.2",
-	"filetype>=1.0.5,<2"
+	"netifaces>=0.10.9",
+	"pylru>=1.2",
+	"rsa>=4.0",
+	"pkginfo>=1.5.0.1",
+	"requests>=2.22.0",
+	"semantic_version>=2.8",
+	"psutil>=5.6.5",
+	"Click>=7",
+	"awesome-slugify>=1.6.5",
+	"feedparser>=5.2.1",
+	"future>=0.18.2",
+	"websocket-client>=0.56",
+	"wrapt>=1.11.2",
+	"emoji>=0.5.4",
+	"frozendict>=1.2",
+	"sentry-sdk>=0.13.2",
+	"filetype>=1.0.5"
 ]
 
 # Python 2 specific requirements
