--- /tmp/floatn.h	2018-06-12 07:31:21.021786078 -0700
+++ /usr/include/bits/floatn.h	2018-06-12 07:27:52.388798074 -0700
@@ -26,13 +26,16 @@
    glibc includes corresponding *f128 interfaces for it.  The required
    libgcc support was added some time after the basic compiler
    support, for x86_64 and x86.  */
-#if (defined __x86_64__							\
-     ? __GNUC_PREREQ (4, 3)						\
-     : (defined __GNU__ ? __GNUC_PREREQ (4, 5) : __GNUC_PREREQ (4, 4)))
-# define __HAVE_FLOAT128 1
-#else
+
+// saa 12 Jun 18: https://devtalk.nvidia.com/default/topic/1023776/-request-add-nvcc-compatibility-with-glibc-2-26/
+
+//#if (defined __x86_64__							\
+//     ? __GNUC_PREREQ (4, 3)						\
+//     : (defined __GNU__ ? __GNUC_PREREQ (4, 5) : __GNUC_PREREQ (4, 4)))
+//# define __HAVE_FLOAT128 1
+//#else
 # define __HAVE_FLOAT128 0
-#endif
+//#endif
 
 /* Defined to 1 if __HAVE_FLOAT128 is 1 and the type is ABI-distinct
    from the default float, double and long double types in this glibc.  */
