--- libSavitar-3.6.0/src/Scene.cpp	2017-11-05 17:24:59.000000000 -0800
+++ libSavitar-3.6.0/src/Scene.cpp	2018-12-17 10:21:05.752019402 -0800
@@ -17,7 +17,7 @@
  */
 
 #include "Scene.h"
-#include "../pugixml/src/pugixml.hpp"
+#include <pugixml.hpp>
 #include <iostream>
 #include <string>
 using namespace Savitar;
--- libSavitar-3.6.0/src/ThreeMFParser.h	2017-11-05 17:24:59.000000000 -0800
+++ libSavitar-3.6.0/src/ThreeMFParser.h	2018-12-17 10:21:05.995017429 -0800
@@ -21,7 +21,7 @@
 
 #include "SavitarExport.h"
 #include "SceneNode.h"
-#include "../pugixml/src/pugixml.hpp"
+#include <pugixml.hpp>
 
 #include <string>
 namespace Savitar
--- libSavitar-3.6.0/src/MeshData.cpp	2017-11-05 17:24:59.000000000 -0800
+++ libSavitar-3.6.0/src/MeshData.cpp	2018-12-17 10:21:06.198015784 -0800
@@ -17,7 +17,7 @@
  */
 
 #include "MeshData.h"
-#include "../pugixml/src/pugixml.hpp"
+#include <pugixml.hpp>
 #include <iostream>
 
 using namespace Savitar;
--- libSavitar-3.6.0/src/SceneNode.cpp	2017-11-05 17:24:59.000000000 -0800
+++ libSavitar-3.6.0/src/SceneNode.cpp	2018-12-17 10:21:05.518021300 -0800
@@ -17,7 +17,7 @@
  */
 
 #include "SceneNode.h"
-#include "../pugixml/src/pugixml.hpp"
+#include <pugixml.hpp>
 #include <iostream>
 using namespace Savitar;
 
--- libSavitar-3.6.0/CMakeLists.txt	2017-11-05 17:24:59.000000000 -0800
+++ libSavitar-3.6.0/CMakeLists.txt	2018-12-17 10:25:33.102916619 -0800
@@ -8,7 +8,7 @@
 option(BUILD_STATIC "Build as a static library" OFF)
 
 
-add_subdirectory(pugixml)
+find_package(pugixml REQUIRED)
 
 if(BUILD_PYTHON)
     set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
